shared/constants/hex.js

```diff
@@ -0,0 +1,2 @@
+/* Represents a zero value in hex */
+export const HEX_ZERO_VALUE = '0x0';
```

Comment:
Could we make this a typescript file? I think i'm going to start pushing for all new files in shared folder to be typescript, simply because they are often leaf node files.